test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -5,6 +5,147 @@
  */
 #include "serializer.h"
 
+#include <stack>
+#include <string>
+
+namespace {
+
+const std::string kCounterNamePrefix = "counter_";
+
+/**
+ * Class that controls id creation for counters. Basically, a variable wrapper
+ * that guarantees variable to be incremented.
+ */
+class CounterIdProvider_ {
+public:
+	/** Returns number of id provided. */
+	std::size_t
+	count()
+	{
+		return id_;
```

Comment:
Why `id_` instead of just `id`?

Reply:
There is a naming convention for private class members in Google (but not only), but actually, there are many naming conventions. Since there is almost no code in C++ within Tarantool, the naming convention can be anything. If you insist on removing the trailing underscore, I will do so.